src/asciidoc/docs/customizations/video.adoc

```diff
@@ -1,15 +1,18 @@
 = Video
 :page-slug: video
+:page-description: Standard VCC for uploading video files with automatic transcoding and formatting.
 
-* {blank}
+* Users can
 //tag::description[]
-Uploaded video is automatically transcoded to be appropriate for playback across devices.
+upload video files.
 //end::description[]
+* Uploaded videos are automatically compressed, transcoded to .mp4 format, and cropped to fit 16:9 portrait for optimal playback across devices.
```

Comment:
I'm not sure where this is defined exactly, but the platform seems to crop videos differently depending on the original aspect ratio. The resulting video is either 16:9 (landscape) or 9:16 (portrait)